integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA3336HostHeaderValuePortCheckTestCase.java

```diff
@@ -49,7 +49,7 @@ public void setEnvironment() throws Exception {
         context = new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN);
         serverConfigurationManager = new ServerConfigurationManager(context);
         String carbonHome = System.getProperty(ServerConstants.CARBON_HOME);
-        File log4jProperties = new File(carbonHome + File.separator + "conf" + File.separator + "log4j.properties");
+        File log4jProperties = new File(carbonHome + File.separator + "conf" + File.separator + "log4j2.properties");
         applyProperty(log4jProperties, "log4j.logger.org.apache.synapse.transport.http.wire", "DEBUG");
```

Comment:
do we need this now

Reply:
yes we still need it because it turns debug logs.